spec.html

```diff
@@ -41121,21 +41121,21 @@ <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
               1. Let _targetBuffer_ be _A_.[[ViewedArrayBuffer]].
               1. Let _elementSize_ be TypedArrayElementSize(_O_).
               1. Let _srcByteOffset_ be _O_.[[ByteOffset]].
-              1. Let _srcByteIndex_ be (_k_ × _elementSize_) + _srcByteOffset_.
+              1. Let _srcByteIndex_ be (_start_ × _elementSize_) + _srcByteOffset_.
               1. Let _targetByteIndex_ be _A_.[[ByteOffset]].
-              1. Let _limit_ be _targetByteIndex_ + (_count_ × _elementSize_).
-              1. Repeat, while _targetByteIndex_ &lt; _limit_,
+              1. Let _endByteIndex_ be _targetByteIndex_ + (_countBytes_ × _elementSize_).
+              1. Repeat, while _targetByteIndex_ &lt; _endByteIndex_,
                 1. Let _value_ be GetValueFromBuffer(_srcBuffer_, _srcByteIndex_, ~uint8~, *true*, ~unordered~).
                 1. Perform SetValueInBuffer(_targetBuffer_, _targetByteIndex_, ~uint8~, _value_, *true*, ~unordered~).
                 1. Set _srcByteIndex_ to _srcByteIndex_ + 1.
                 1. Set _targetByteIndex_ to _targetByteIndex_ + 1.
             1. Else,
               1. Let _n_ be 0.
-              1. Repeat, while _k_ &lt; _final_,
-                1. Let _Pk_ be ! ToString(𝔽(_k_)).
+              1. Repeat, while _start_ &lt; _end_,
+                1. Let _Pk_ be ! ToString(𝔽(_start_)).
                 1. Let _kValue_ be ! Get(_O_, _Pk_).
                 1. Perform ! Set(_A_, ! ToString(𝔽(_n_)), _kValue_, *true*).
-                1. Set _k_ to _k_ + 1.
+                1. Set _start_ to _start_ + 1.
```

Comment:
I'd prefer not to increment a variable named `start` or anything like it. We use _k_ as an index in >50 places in the spec; seems fine to keep using it here.

Reply:
Done, and updated commit message.